src/libs/actions/IOU.ts

```diff
@@ -3122,6 +3127,7 @@ function createSplitsAndOnyxData(
     existingSplitChatReportID = '',
     billable = false,
     iouRequestType: IOURequestType = CONST.IOU.REQUEST_TYPE.MANUAL,
+    splitPayerAccountIDs: number[] = [],
```

Comment:
Let's make sure we use this param and send it to the backend here https://github.com/Expensify/App/blob/21ee1a3772253538bba74b2bc655213dc7bdb68a/src/libs/actions/IOU.ts#L3458